tmx_tileset.go

```diff
@@ -76,8 +78,10 @@ type Terrain struct {
 type TilesetTile struct {
 	// The local tile ID within its tileset.
 	ID uint32 `xml:"id,attr"`
-	// The type of the tile. Refers to an object type and is used by tile objects. (optional) (since 1.0)
+	// The type of the tile. Refers to an object type and is used by tile objects. (optional) (since 1.0, until 1.8)
 	Type string `xml:"type,attr"`
```

Comment:
Can you add deprecated comment, something like:
```go
//
// Deprecated: replaced by Class since 1.9
```

Reply:
Done.